### PR TITLE
Pioneer 16 Deploy/Recover updates

### PR DESCRIPTION
**Pull request by snwhite**

### CODE REVIEW

File: deployment/CP03ISSM_Deploy.csv [hunk 1]
```diff
@@ -341,65 +341,65 @@ AR39,,AR48,,CP03ISSM-SBD11-06-METBKA000,11,1,2019-09-26T13:50:00,2020-11-01T13:1
...
+AR52,,,,CP03ISSM-RID27-04-DOSTAD000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-DOSTAD-00131,40.3671,-70.881567,,7,93,,,
+AR52,,,,CP03ISSM-SBC11-00-CPMENG000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGCON-BCPM01-50015,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD11-00-DCLENG000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGCON-BDCL11-50015,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD11-01-MOPAK0000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-MOPAK0-43257,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD11-02-HYDGN0000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-HYDGN0-42316,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-METLGR-00008,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD12-00-DCLENG000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGCON-BDCL12-50015,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD12-03-HYDGN0000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-HYDGN0-41917,40.3671,-70.881567,,0,93,,,
+AR52,,,,CP03ISSM-SBD12-04-PCO2AA000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013,,CGINS-PCO2AA-34231,40.3671,-70.881567,,0,93,,,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/CP03ISSM_Deploy.csv:405]: The ISSM-1 updates look good too.

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 2]
```diff
@@ -65,9 +65,15 @@ TS20200613,,MZ20200731,,CP05MOAS-GL339-02-FLORTM000,11,1,2020-06-13T20:09:37,202
...
+TS20210131,,AR52,,CP05MOAS-GL339-05-PARADM000,12,1,2021-01-31T16:40:00,2021-03-30T21:30:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.09157,-71.16818,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-00-ENG000000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.07595,-70.383683,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-01-ADCPAM000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-50865,40.07595,-70.383683,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-02-FLORTM000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04472,40.07595,-70.383683,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-03-CTDGVM000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09338,40.07595,-70.383683,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-04-DOSTAM000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00161,40.07595,-70.383683,,200,200,EB,,
+AR52,,,,CP05MOAS-GL339-05-PARADM000,13,1,2021-04-02T13:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.07595,-70.383683,,200,200,EB,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL339_Deploy.csv:79]: looks good

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 3]
```diff
@@ -71,9 +71,9 @@ TS20200819,,AR48,,CP05MOAS-GL376-02-FLORTM000,12,1,2020-08-19T17:11:00,2020-11-0
...
\ No newline at end of file
+TS20210131,,AR52,,CP05MOAS-GL376-00-ENG000000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210131,,AR52,,CP05MOAS-GL376-01-ADCPAM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210131,,AR52,,CP05MOAS-GL376-02-FLORTM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03116,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210131,,AR52,,CP05MOAS-GL376-03-CTDGVM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09089,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210131,,AR52,,CP05MOAS-GL376-04-DOSTAM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00543,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210131,,AR52,,CP05MOAS-GL376-05-PARADM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.92755,-71.1765,,1000,1000,SS-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL376_Deploy.csv:79]: looks good

File: deployment/CP05MOAS-PG564_Deploy.csv [hunk 4]
```diff
@@ -13,10 +13,10 @@ WJ20170815,,TA20171007,,CP05MOAS-PG564-03-FLORTM000,2,1,2017-08-17T10:20:00,2017
...
+AR52,,,,CP05MOAS-PG564-00-ENG000000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGCON-GLDRCP-00564,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-01-CTDGVM000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09295,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-02-DOSTAM000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00432,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-03-FLORTM000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-04089,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-04-FLORTO000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01347,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-05-NUTNRM000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00662,40.1000167,-71.00145,,200,200,CNPM,,
+AR52,,,,CP05MOAS-PG564-06-PARADM000,3,1,2021-04-03T18:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50202,40.1000167,-71.00145,,200,200,CNPM,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-PG564_Deploy.csv:22]: looks good

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 5]
```diff
@@ -20,10 +20,10 @@ TS20200819,,AR48,,CP05MOAS-PG583-03-FLORTM000,3,1,2020-08-19T19:25:00,2020-10-29
...
+AR52,,,,CP05MOAS-PG583-00-ENG000000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGCON-GLDRCP-00583,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-01-CTDGVM000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-CTDGVM-09301,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-02-DOSTAM000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-DOSTAM-00539,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-03-FLORTM000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTM-04101,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-04-FLORTO000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-FLORTO-01342,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-05-NUTNRM000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-NUTNRM-00660,40.097433,-71.000567,,200,200,ISPM,,
+AR52,,,,CP05MOAS-PG583-06-PARADM000,4,1,2021-04-03T19:13:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00583,CGINS-PARADM-50217,40.097433,-71.000567,,200,200,ISPM,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-PG583_Deploy.csv:29]: looks good

**Review by rtravis-whoi — APPROVED** [review]
glider csvs look good; already approved SM csvs.

File: deployment/CP05MOAS-PG564_Deploy.csv (hunk 4 above)
Comment (IDCalTech) [reply deployment/CP05MOAS-PG564_Deploy.csv:22]: looks good

File: deployment/CP05MOAS-PG583_Deploy.csv (hunk 5 above)
Comment (IDCalTech) [reply deployment/CP05MOAS-PG583_Deploy.csv:29]: looks good

**Review by IDCalTech — APPROVED** [review]
everything checks out

File: deployment/CP01CNSM_Deploy.csv [hunk 6]
```diff
@@ -407,71 +407,71 @@ AR39,,AR48,,CP01CNSM-SBD12-04-PCO2AA000,12,1,2019-09-27T18:30:00,2020-11-06T13:1
...
+AR52,,,,CP01CNSM-SBD11-01-MOPAK0000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-MOPAK0-11843,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD11-02-HYDGN0000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-HYDGN0-42330,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD11-06-METBKA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-METLGR-00010,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-00-DCLENG000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGCON-BDCL12-20001,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-03-HYDGN0000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-HYDGN0-Z5971,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-04-PCO2AA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-PCO2AA-34223,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-05-WAVSSA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-WAVSSA-05311,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-06-METBKA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-METLGR-00017,40.132833,-70.778,,0,135,,,
+AR52,,,,CP01CNSM-SBD12-08-FDCHPA000,14,1,2021-03-31T15:36:00,,CGMCP-01CNSM-00014,,CGINS-FDCHPA-64401,40.132833,-70.778,,-5,135,,,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply deployment/CP01CNSM_Deploy.csv:477]: looks good

File: deployment/CP03ISSM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:405]: looks good

File: deployment/CP04OSSM_Deploy.csv [hunk 7]
```diff
@@ -340,67 +340,67 @@ AR39,,AR48,,CP04OSSM-SBD11-06-METBKA000,11,1,2019-09-29T14:46:00,2020-11-08T13:1
...
+AR52,,,,CP04OSSM-SBD11-01-MOPAK0000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-MOPAK0-42298,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD11-02-HYDGN0000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-HYDGN0-49886,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD11-06-METBKA000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-METLGR-00037,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD11-08-FDCHPB000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-FDCHPB-03902,39.937217,-70.8897,,-5,448,Test Upgraded FDCHP w/ R3-50,,
+AR52,,,,CP04OSSM-SBD12-00-DCLENG000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGCON-BDCL12-20003,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD12-03-HYDGN0000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-HYDGN0-49857,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD12-04-PCO2AA000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-PCO2AA-33154,39.937217,-70.8897,,0,448,,,
+AR52,,,,CP04OSSM-SBD12-08-FDCHPA000,13,1,2021-03-30T15:02:00,,CGMCP-04OSSM-00013,,CGINS-FDCHPA-44903,39.937217,-70.8897,,-5,448,Test Standard FDCHP w/ WindMaster Pro,,
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:406]: looks good

File: deployment/CP05MOAS-GL339_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/CP05MOAS-GL339_Deploy.csv:79]: looks good

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/CP05MOAS-GL376_Deploy.csv:79]: looks good

File: deployment/CP05MOAS-PG564_Deploy.csv (hunk 4 above)
Comment (cdobs) [inline deployment/CP05MOAS-PG564_Deploy.csv:22]: looks good

File: deployment/CP05MOAS-PG583_Deploy.csv (hunk 5 above)
Comment (cdobs) [inline deployment/CP05MOAS-PG583_Deploy.csv:29]: looks good

**Review by cdobs — APPROVED** [review]
Leg 1 deploy location/times and SN updates reviewed, looks good

**Pull request merged**